Test failure:
```
======================================================================
FAIL: test_domain_reindexer (testapps.test_pillowtop.tests.test_reindexer.PillowtopReindexerTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/testapps/test_pillowtop/tests/test_reindexer.py", line 44, in test_domain_reindexer
    self.assertEqual(1, results.total)
AssertionError: 1 != 2
```

--- a/corehq/apps/domain/tests/test_utils.py
+++ b/corehq/apps/domain/tests/test_utils.py
@@ -4,5 +4,5 @@ from corehq.util.test_utils import unit_testing_only
 
 @unit_testing_only
 def delete_all_domains():
-    domains = Domain.get_all()
+    domains = list(Domain.get_all())
     Domain.bulk_delete(domains)

